common/src/main/java/com/discordsrv/common/config/main/sync/BanSyncConfig.java

```diff
@@ -39,6 +39,12 @@ public BanSyncConfig() {
     @Order(-10)
     public long serverId = 0L;
 
+    @Comment("On a player's first join to the server, prefer the Discord side for tie-breaking")
+    public boolean preferDiscordTieBreakerOnFirstJoin = true;
+
+    @Comment("When a player links their account, prefer the Discord side for tie-breaking")
+    public boolean preferDiscordTieBreakerOnAccountLink = true;
```

Comment:
This should already be possible to configure via `ban-sync.tie-breakers.link` (or along those lines)